src/client/app/components/MultiSelectComponent.jsx

```diff
@@ -32,9 +32,11 @@ export default class MultiSelectComponent extends React.Component {
 	}
 
 	render() {
+		let multi = true;
```

Comment:
Why do you have logic dealing with multi here?I think you can remove all your changes here regarding multi.